### DISCUSSION TITLE
Error when saving

### DISCUSSION THREAD

**Discussion by melnychuk1985** [wintercms/winter#332] [Q&A] [answered]
When I try to save the default.htm file that was not changed or modified, it gives an error and does not save

![screen_2021-10-10](https://user-images.githubusercontent.com/28691146/136673870-c2a8b35b-a5cb-41b3-9983-cb8965224048.png)

**Comment by LukeTowers** [comment, marked as answer]
Seems like there's an error with either your permissions or perhaps mod_security. See https://octobercms.com/support/article/ke-1.

**Reply by melnychuk1985** [reply]
Thank you for not leaving my question unanswered!
